src/app/shared/script.service.ts

```diff
@@ -59,6 +59,11 @@ export class ScriptService {
                         resolve({script: name, loaded: true, status: 'loaded'});
                     };
                 }
+                // window binding for mouseflow script
+                if (script.id === 'mouseflow') {
+                  script.innerText = 'window._mfq = window._mfq || [];'
```

Comment:
Nice solution! What if we moved the innerText value to the "store" object, so any scripts in the future could include an innerText?

Reply:
Really good idea! Update coming...

Reply:
@codiak Updated. Let me know what you think 👍